.github/workflows/main.yml

```diff
@@ -8,70 +8,80 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [py36, py37, py38, py39, py310]
+        name: ["py36", "py37", "py38", "py39", "py310", "py311-dev"]
```

Comment:
After this change, is this name still needed? It seems never referred.

Reply:
`name` is used in the GitHub UI, I've updated it though to be simpler.

A